openupgrade_scripts/scripts/account/18.0.1.3/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,377 @@
+account      / account.cash.rounding    / profit_account_id (many2one)  : needs conversion to v18-style company dependent
+# DONE: post-migration: used openupgrade_180.convert_company_dependent
+
+account      / account.group            / parent_path (char)            : DEL
+# NOTHING TO DO
+
+account      / account.journal          / autocheck_on_post (boolean)   : NEW hasdefault: default
+# DONE: pre-migration: pre-created and filled
+
+account      / account.journal          / sale_activity_note (text)     : DEL
+account      / account.journal          / sale_activity_type_id (many2one): DEL relation: mail.activity.type
+account      / account.journal          / sale_activity_user_id (many2one): DEL relation: res.users
+# NOTHING TO DO: https://github.com/odoo/odoo/commit/ab3c81b680a1019ff33aabf87cf533af7bfe8cfa
+
+account      / account.journal          / secure_sequence_id (many2one) : DEL relation: ir.sequence
+# NOTHING TO DO: https://github.com/odoo/odoo/commit/9a1b320255766f4c7c72e9bbf50c1d50615ab397
+
+account      / account.journal          / type (selection)              : selection_keys is now '['bank', 'cash', 'credit', 'general', 'purchase', 'sale']' ('['bank', 'cash', 'general', 'purchase', 'sale']')
+# NOTHING TO DO: new 'credit' option
+
+account      / account.lock_exception   / active (boolean)              : NEW hasdefault: default
+account      / account.lock_exception   / company_id (many2one)         : NEW relation: res.company, required
+account      / account.lock_exception   / company_lock_date (date)      : NEW
+account      / account.lock_exception   / end_datetime (datetime)       : NEW
+account      / account.lock_exception   / lock_date (date)              : NEW
+account      / account.lock_exception   / lock_date_field (selection)   : NEW required, selection_keys: ['fiscalyear_lock_date', 'purchase_lock_date', 'sale_lock_date', 'tax_lock_date']
+account      / account.lock_exception   / reason (char)                 : NEW
+account      / account.lock_exception   / user_id (many2one)            : NEW relation: res.users, hasdefault: default
+# NOTHING TO DO? (new model)
+
+account      / account.move             / amount_untaxed_in_currency_signed (float): NEW isfunction: function, stored
+# DONE: pre-migration: pre-created and filled
+
+account      / account.move             / audit_trail_message_ids (one2many): NEW relation: mail.message
+# NOTHING TO DO
+
+account      / account.move             / checked (boolean)             : NEW
+account      / account.move             / to_check (boolean)            : DEL
+# DONE: pre-migration: pre-created and filled
+
+account      / account.move             / invoice_currency_rate (float) : NEW isfunction: function, stored
+l10n_dk_bookkeeping / account.move             / l10n_dk_currency_rate_at_transaction (float): DEL
+# DONE: pre-migration: rename field
```

Comment:
this shouldn't be here in the first place, right? Should be taken care of by l10n_dk

Reply:
`l10n_dk_bookkeeping` is merged into `account`.